monitor.py

```diff
@@ -136,6 +136,7 @@ def timestamp(self):
         timestamp, new = get_one_or_create(
             self._db, Timestamp,
             service=self.service_name,
+            service_type=Timestamp.MONITOR_TYPE,
```

Comment:
Changing this method ensures that any time a Monitor finds 'its' timestamp it's using the appropriate `service_type`.